### ISSUE TITLE
Array field in list collection throws "template is not defined" error

### ISSUE THREAD

**Issue by ScottHamper** [getgrav/grav-plugin-admin#1926]
## Summary
I have a module for displaying a list of contacts, and each contact has an array field for their job titles. Everyone has at least one title, but some people have two or even three. When adding a new contact, if I input something into the title field, I will receive a "template is not defined" error upon saving (and the page will not save). However, if I leave the title blank, I can add the new contact without any issue and then successfully edit the titles field later. Even in this latter scenario, if I then drag/drop list items to re-order them, I either get the "template is not defined" error again (when the frontmatter uses numbers for the array keys), or the titles get mixed around after the move completes (when the frontmatter uses dashes for the array "keys").

Removing the array field from the blueprint "resolves" the issues mentioned above.

## Steps to Reproduce
1. Download the [attached example blueprint](https://github.com/getgrav/grav-plugin-admin/files/4919562/contacts.txt) and add it to a Grav site. (Rename to .yaml, since GitHub doesn't allow uploading them as attachments)
2. Create a new contacts module.
3. Add a contact item to the list, filling in both the "name" and "titles" fields.
4. Save.

## Expected Result
The module saves successfully.

## Actual Result
`An exception has been thrown during the rendering of a template ("Template "modular/contacts.html.twig" is not defined.").`

**Comment by rhukster** [comment]
I'm pretty sure this is an issue with an `array` inside a `field`. Multiple dynamic fields get tricky. How about trying a commalist?

```
            .title:
              type: selectize
              size: large
              label: Titles
              classes: fancy
              validate:
                type: commalist
```

just to test...

**Comment by w00fz** [comment]
I believe if this was an issue in Admin 1.9, it is now resolved in Admin 1.10. I was able to successfully save different inputs (see screenshot) with Grav 1.10.

<img width="514" alt="test ) | Grav 2020-11-20 14-55-55" src="https://user-images.githubusercontent.com/11734/99857710-b638a900-2b40-11eb-8b26-86f89a0dd828.png">

The error you are reporting seems to be pointing to the fact that you don't have a template `modular/contacts.html.twig` which is required for rendering your custom page type.
